From efd84e3eb345104e62f82c7566fb1b42a04d5d11 Mon Sep 17 00:00:00 2001
From: Mat Booth <mat.booth@gmail.com>
Date: Thu, 4 Apr 2024 15:53:53 +0100
Subject: [PATCH 1/2] Update aes version

---
 Cargo.toml            | 2 +-
 src/authentication.rs | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index 9b8b15f..ebedc0d 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -20,7 +20,7 @@ license = "MIT"
 repository = "https://github.com/librespot-org/librespot"
 
 [dependencies.aes]
-version = "0.6"
+version = "0.7"
 
 [dependencies.base64]
 version = "0.13"
diff --git a/src/authentication.rs b/src/authentication.rs
index 3c188ec..353dc3a 100644
--- a/src/authentication.rs
+++ b/src/authentication.rs
@@ -1,6 +1,6 @@
 use std::io::{self, Read};
 
-use aes::Aes192;
+use aes::{Aes192, BlockDecrypt};
 use byteorder::{BigEndian, ByteOrder};
 use hmac::Hmac;
 use pbkdf2::pbkdf2;
-- 
2.41.0

