From c02215d0e486e01ce3d282426f8c8b67831cdab2 Mon Sep 17 00:00:00 2001
From: Mat Booth <mat.booth@gmail.com>
Date: Fri, 5 Apr 2024 12:57:50 +0100
Subject: [PATCH 1/5] Update alsa version

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 13ae6b2..b06620d 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -49,7 +49,7 @@ assets = [
 ]
 
 [dependencies.alsa]
-version = "0.7"
+version = "0.8"
 optional = true
 
 [dependencies.chrono]
-- 
2.41.0

