From 98ead6700e18ade94d62d662c9a1389c8fc47c24 Mon Sep 17 00:00:00 2001
From: Mat Booth <mat.booth@gmail.com>
Date: Mon, 2 Dec 2024 12:29:00 +0000
Subject: [PATCH 3/3] Update priority-queue version

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index ac0083d..836ae31 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -94,7 +94,7 @@ features = ["hmac"]
 default-features = false
 
 [dependencies.priority-queue]
-version = "1.2"
+version = "2.1"
 
 [dependencies.protobuf]
 version = "2.14.0"
-- 
2.47.0

