From cd360b5a35b8a134f99c98a314f13c35b83fbb7c Mon Sep 17 00:00:00 2001
From: Mat Booth <mat.booth@gmail.com>
Date: Tue, 21 Oct 2025 13:34:23 +0100
Subject: [PATCH] use avahi backend by default

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 3ea4308..833dd25 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -28,7 +28,7 @@ repository = "https://github.com/librespot-org/librespot"
 
 [features]
 default = [
-    "with-libmdns",
+    "with-avahi",
     "native-tls",
 ]
 native-tls = ["librespot-core/native-tls"]
-- 
2.51.0

