From 29392f19109aeb454f41c6c5611adce39d04262a Mon Sep 17 00:00:00 2001
From: Mat Booth <mat.booth@gmail.com>
Date: Tue, 21 Oct 2025 14:00:10 +0100
Subject: [PATCH] switch to alsa backend by default

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 4de3abe..c22cea8 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -29,7 +29,7 @@ repository = "https://github.com/librespot-org/librespot"
 [features]
 alsa-backend = ["dep:alsa"]
 default = [
-    "rodio-backend",
+    "alsa-backend",
     "native-tls",
 ]
 gstreamer-backend = [
-- 
2.52.0

