From 7e166ca303b571c4442f9ec3447807b6dd03d57f Mon Sep 17 00:00:00 2001
From: eladyn <eladyn@protonmail.com>
Date: Wed, 19 Nov 2025 03:48:01 +0100
Subject: [PATCH] dbus_mpris: fix track indexing for OpenUri

Fixes #1377.
---
 src/dbus_mpris.rs | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/src/dbus_mpris.rs b/src/dbus_mpris.rs
index e0f5619..8a8cfe8 100644
--- a/src/dbus_mpris.rs
+++ b/src/dbus_mpris.rs
@@ -17,7 +17,7 @@ use librespot_connect::{LoadContextOptions, LoadRequest, LoadRequestOptions, Spi
 use librespot_core::{Session, SpotifyUri};
 use librespot_metadata::audio::AudioItem;
 use librespot_playback::player::PlayerEvent;
-use log::{debug, error, warn};
+use log::{debug, warn};
 use std::convert::TryFrom;
 use std::{
     collections::HashMap,
@@ -751,7 +751,10 @@ fn register_player_interface(
                     Ok::<_, librespot_core::Error>(match uri {
                         SpotifyUri::Track { .. } => {
                             let track = Track::get(&session, &uri).await?;
-                            (track.number as u32, track.album.id.to_uri()?)
+                            if track.number == 0 {
+                                warn!("track.number is not expected to be 0- but 1-indexed");
+                            }
+                            ((track.number - 1).max(0) as u32, track.album.id.to_uri()?)
                         }
                         SpotifyUri::Album { .. }
                         | SpotifyUri::Artist { .. }
-- 
2.55.0

