From accfc05014c9f65c6634f4ee48f5d085c0bd19f9 Mon Sep 17 00:00:00 2001
From: Mat Booth <mat.booth@gmail.com>
Date: Mon, 20 Oct 2025 14:12:02 +0100
Subject: [PATCH 2/2] switch to syslog 6

---
 Cargo.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index a96f921..78628c5 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -107,7 +107,7 @@ version = "5.0.0"
 [dependencies.fern]
 version = "0.7.0"
 features = [
-    "syslog-7",
+    "syslog-6",
     "colored",
 ]
 
@@ -199,7 +199,7 @@ features = ["bindgen"]
 version = "0.5"
 
 [target."cfg(unix)".dependencies.syslog]
-version = "7"
+version = "6"
 
 [profile.release]
 lto = true
-- 
2.51.0

