From bc36c9cee1e5ed52a1428074cd98f52ad5fb4142 Mon Sep 17 00:00:00 2001
From: Mat Booth <mat.booth@gmail.com>
Date: Tue, 21 Oct 2025 10:54:38 +0100
Subject: [PATCH 2/3] fix tungstenite dependency

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 4b9d815..75058a7 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -247,7 +247,7 @@ features = [
 default-features = false
 
 [dependencies.tokio-tungstenite]
-version = "0.28"
+version = "0.26"
 default-features = false
 
 [dependencies.tokio-util]
-- 
2.51.0

