From f40b667a9e87dddd8167f0094bb86aabb855c9d3 Mon Sep 17 00:00:00 2001
From: Mat Booth <mat.booth@gmail.com>
Date: Tue, 21 Oct 2025 10:59:33 +0100
Subject: [PATCH 3/3] fix vergen dependency

---
 Cargo.toml |  6 +++---
 build.rs   | 21 ++++++---------------
 2 files changed, 9 insertions(+), 18 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index 75058a7..b438ff3 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -275,7 +275,7 @@ version = "0.9"
 [build-dependencies.rand_distr]
 version = "0.5"
 
-[build-dependencies.vergen-gitcl]
-version = "1.0"
-features = ["build"]
+[build-dependencies.vergen]
+version = "8"
+features = ["build", "git", "gitcl"]
 default-features = false
diff --git a/build.rs b/build.rs
index 4413d2f..3d96fe5 100644
--- a/build.rs
+++ b/build.rs
@@ -1,20 +1,12 @@
 use rand::Rng;
 use rand_distr::Alphanumeric;
-use vergen_gitcl::{BuildBuilder, Emitter, GitclBuilder};
+use vergen::EmitBuilder;
 
-fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let gitcl = GitclBuilder::default()
-        .sha(true) // outputs 'VERGEN_GIT_SHA', and sets the 'short' flag true
-        .commit_date(true) // outputs 'VERGEN_GIT_COMMIT_DATE'
-        .build()?;
-
-    let build = BuildBuilder::default()
-        .build_date(true) // outputs 'VERGEN_BUILD_DATE'
-        .build()?;
-
-    Emitter::default()
-        .add_instructions(&build)?
-        .add_instructions(&gitcl)?
+fn main() {
+    EmitBuilder::builder()
+        .build_date() // outputs 'VERGEN_BUILD_DATE'
+        .git_sha(true) // outputs 'VERGEN_GIT_SHA', and sets the 'short' flag true
+        .git_commit_date() // outputs 'VERGEN_GIT_COMMIT_DATE'
         .emit()
         .expect("Unable to generate the cargo keys!");
     let build_id = match std::env::var("SOURCE_DATE_EPOCH") {
@@ -27,5 +19,4 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     };
 
     println!("cargo:rustc-env=LIBRESPOT_BUILD_ID={build_id}");
-    Ok(())
 }
-- 
2.51.0

