From 1901ed893e0757b768d0897c582302d70ff8fd6f Mon Sep 17 00:00:00 2001
From: Mat Booth <mat.booth@gmail.com>
Date: Sun, 19 Oct 2025 13:33:09 +0100
Subject: [PATCH 2/2] relax multimap version

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 1ee7390..b923f55 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -36,7 +36,7 @@ features = ["link-local"]
 version = "0.4"
 
 [dependencies.multimap]
-version = "0.10.0"
+version = "0.8.3"
 
 [dependencies.rand]
 version = "0.8"
-- 
2.51.0

